gazebo/Dockerfile

```diff
@@ -10,39 +10,51 @@ ARG IGN_GUI_PLUGIN_PATH_USER=/models_database/plugins/gui
 ARG IGN_GAZEBO_RESOURCE_PATH=/models_database/my_models
 ARG APT_REPOSITORY="https://artifacts.aws.cloud.mov.ai/repository"
 
+ARG LATEST_COMPONENTS=( )
+
 ENV MOVAI_ENV="release"
 
 SHELL ["/bin/bash", "-o", "pipefail", "-c"]
 
-# Copy runtime scripts
+# Copy runtime scripts and setup files
 COPY ./gazebo/movai-entrypoint.sh /usr/local/bin/movai-entrypoint.sh
+COPY ./gazebo/update-collection-fortress-to-latest.bash /usr/local/bin/update-collection-fortress-to-latest.bash
```

Comment:
do these scripts need to be kept for runtime or they are only needed for build ?